packages/frameworks/vue/src/presence/presence.tsx

```diff
@@ -19,20 +19,6 @@ export const Presence = defineComponent<PresenceProps>(
   },
   {
     name: 'Presence',
-    props: {
-      present: {
-        type: Boolean as PropType<UsePresenceProps['present']>,
-        default: undefined,
-      },
-      lazyMount: {
-        type: Boolean as PropType<UsePresenceProps['lazyMount']>,
-        default: undefined,
-      },
-      unmountOnExit: {
-        type: Boolean as PropType<UsePresenceProps['unmountOnExit']>,
-        default: undefined,
-      },
-    },
-    emits,
```

Comment:
As we use Function Signature to define component, I don't think we need to declare emit to get the type hint of event, because they already exist in the definition of zag

Reply:
Good work with cleaning `props` in the component.

I'm not sure if `emits` are needed in components